man/linters.Rd

```diff
@@ -17,15 +17,15 @@ Documentation for linters is structured into tags to allow for easier discovery.
 \section{Tags}{
 The following tags exist:
 \itemize{
-\item{\link[=best_practices_linters]{best_practices} (17 linters)}
+\item{\link[=best_practices_linters]{best_practices} (19 linters)}
 \item{\link[=common_mistakes_linters]{common_mistakes} (5 linters)}
 \item{\link[=configurable_linters]{configurable} (16 linters)}
 \item{\link[=consistency_linters]{consistency} (7 linters)}
 \item{\link[=correctness_linters]{correctness} (7 linters)}
 \item{\link[=default_linters]{default} (25 linters)}
 \item{\link[=efficiency_linters]{efficiency} (4 linters)}
-\item{\link[=package_development_linters]{package_development} (9 linters)}
-\item{\link[=readability_linters]{readability} (23 linters)}
+\item{\link[=package_development_linters]{package_development} (11 linters)}
+\item{\link[=readability_linters]{readability} (25 linters)}
```

Comment:
Why +2? Did we forget a `document()` in an earlier PR?
Especially weird because the tag Rds only get one additional line...

Reply:
😬 might be weird merging, either in this PR or in a recent one. I'll re-document() before next push.

Reply:
nothing changed... here's the `package_development` list, which looks right. HEAD has 10 listed: https://github.com/r-lib/lintr/blob/585e7cd3e0e4a3aa8f78fdf0242a908ff4cfd65e/man/package_development_linters.Rd#L15-L24